examples/okta_bookmark_app/basic_updated.tf

```diff
@@ -0,0 +1,22 @@
+resource "okta_user" "user" {
+  admin_roles = ["APP_ADMIN", "USER_ADMIN"]
+  first_name  = "TestAcc"
+  last_name   = "blah"
+  login       = "test-acc-%[1]d@testing.com"
+  email       = "test-acc-%[1]d@testing.com"
+  status      = "ACTIVE"
```

Comment:
This param defaults to ACTIVE if you wanted to drop it from the example for simplicity